### ISSUE TITLE
Eclipse plugin should run on multiple files when multiple files are selected

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#189]
```
What steps will reproduce the problem?
1.  Start up eclipse with the plugin installed.
2.  Select a project with at least two java files in the same directory.  These 
files should have failing assignments (say null assigned to @NonNull values) 
for a given checker in the same directory.
3.  Select both files.
4.  Right click (making sure to preserver the selection) and run the checker 
that should produce errors.

What is the expected output? What do you see instead?
The console should output checking errors for both files and the files should 
have error markers affixed to any failed checks.  Only errors for the first 
file are output but likely we should check both files in this instance.

This problem can be solved in: CheckerHandler.element.  This at the moment only 
selects the first element in a section.  I would do this without reporting an 
issue except the ability to select a file and one or more packages can make 
this slightly more than trivial.
```

Original issue reported on code.google.com by `Jonathan...@gmail.com` on 30 Dec 2012 at 10:25
